webapp/src/js/controllers/inbox.js

```diff
@@ -678,10 +679,19 @@ var feedback = require('../modules/feedback'),
     // https://github.com/medic/medic-webapp/issues/2927
     $rootScope.$on('$stateChangeStart', closeDropdowns);
 
+    DatabaseConnectionMonitor.onDatabaseClosed().then(function() {
```

Comment:
Feel good about the naming for this service?

Reply:
Yeah it makes sense to me.